### PR TITLE
Fixed typos in README

### PR DESCRIPTION
**Pull request by vicky002**
Hi,

I fixed few small typos in README.md file. Please check once and let me know if there are any issues. 

Thanks! 

Also, it would be good if we specify the latest version of dependencies in requirements.txt file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ _class_ luminol.modules.correlation_result.**CorrelationResult**
 ```python
 self.coefficient: # correlation coefficient.
 self.shift: # the amount of shift needed to get the above coefficient.
-self.shifted_coefficient: # a correlation coefficient with shift taken into account.
+self.shifted_coefficient: # a correlation coefficient with shift is taken into account.
```
Comment (RiteshMaheshwari) [inline README.md]: The original text is correct in this case.

**Comment by RiteshMaheshwari** [comment]
@vicky002 thanks for the pull request. If you can fix the problems, I will merge.

**Comment by vicky002** [comment]
Hi @RiteshMaheshwari sir, 

Please check and let me know if there are any further changes.

**Comment by RiteshMaheshwari** [comment]
Thanks @vicky002!

**Comment by vicky002** [comment]
Whoah! Thank you sir. Working on the code, more commits soon! :)

**Pull request merged**